Add readme, standard schematic, hardware design discussion

robpl4788 opened the pull request:
Added a readme to the wiki with quick links to some useful pages

Did some slight refactoring and renaming to remove crawler specific terminology from general guides

Added the standard schematic

Added a hardware design guide which gives some insight on hardware choices when designing custom esc's. Probably worth proofreading.

VitroidFPV commented:
Why was `/genera/docs/docs.md` deleted? That currently breaks the CTA and nav bar links.

robpl4788 commented:
> Why was `/docs/general/docs.md` deleted? That currently breaks the CTA and nav bar links.

Does it? I didn't realise. I deleted it because I moved all that content to the readme, which I think is a better location for it
